governance.md

```diff
@@ -0,0 +1,393 @@
+sponsor, may hold project trademarks and other intellectual property, helps
+manage project donations and acts as a parent legal entity. NumFOCUS is the
+only legal entity that has a formal relationship with the project (see
+Institutional Partners section below).
+
+
+Governance
+==========
+
+This section describes the governance and leadership model of The Project.
+
+The foundations of Project governance are:
+
+-   Openness & Transparency
+-   Active Contribution
+-   Institutional Neutrality
+
+Traditionally, Project leadership was provided by a BDFL (Matthew Rocklin) and
+a subset of Contributors, called Core Developers, whose active and consistent
+contributions have been recognized by their receiving ownership rights over the
+GitHub organization. In general all Project decisions are made through
+consensus among the Core Developers with input from the Community. The BDFL
+can, but rarely chooses to, override the Core Developers and make a final
+decision on a matter.
+
+While this approach has served us well, as the Project grows and faces more
+legal and financial decisions and interacts with other institutions, we see a
+need for a more formal governance model. Moving forward The Project leadership
+will consist of a Steering Council and BDFL. We view this governance model as
+the formalization of what we are already doing, rather than a change in
+direction.
+
+
+BDFL
+----
+
+The Project will have a BDFL (Benevolent Dictator for Life), who is currently
+Matthew Rocklin. As Dictator, the BDFL has the authority to make all final
+decisions for The Project. As Benevolent, the BDFL, in practice chooses to
+defer that authority to the consensus of the community discussion channels and
+the Steering Council (see below). It is expected, and in the past has been the
+case, that the BDFL will only rarely assert their final authority. Because
+rarely used, we refer to BDFL’s final authority as a “special” or “overriding”
+vote. When it does occur, the BDFL override typically happens in situations
+where there is a deadlock in the Steering Council or if the Steering Council
+asks the BDFL to make a decision on a specific matter. To ensure the
+benevolence of the BDFL, The Project encourages others to fork the project if
+they disagree with the overall direction the BDFL is taking. The BDFL is chair
+of the Steering Council (see below) and may delegate their authority on a
+particular decision or set of decisions to any other Council member at their
+discretion.
+
+The BDFL can appoint their successor, but it is expected that the Steering
+Council would be consulted on this decision. If the BDFL is unable to appoint a
+successor, the Steering Council will make a suggestion or suggestions to the
+Main NumFOCUS Board. While the Steering Council and Main NumFOCUS Board will
+work together closely on the BDFL selection process, the Main NumFOCUS Board
+will make the final decision.
```

Comment:
To be honest with you, I would rather we say NumFOCUS Board will oversee an election from a group of stakeholders.

Reply:
Yeah, it does feel a little strange. What would be the "group of stakeholders" in this case? The Steering Council? Something else?

Reply:
@aterrel, could you please clarify what you mean by "group of stakeholders"?

Reply:
Have pushed a commit to change this to the Steering Council will make this decision. Thanks for the feedback.